packages/zowe-explorer/src/job/actions.ts

```diff
@@ -406,16 +408,6 @@ async function deleteMultipleJobs(
         await errorHandling(userMessage);
     }
 }
-
-export async function openMultipleJcl(jobContentList: string[]) {
-    let docContent: string = "";
-    for await (const jobContent of jobContentList) {
-        docContent += jobContent + "\n";
-    }
-    const jclDoc = await vscode.workspace.openTextDocument({ language: "jcl", content: docContent });
-    await vscode.window.showTextDocument(jclDoc);
-}
-
 // class Spool extends Job {
 //     constructor(
 //         label: string,
```

Comment:
curious if this commented-out class be removed as well ?

Reply:
Thanks I didn't changed the lines but also don't see any reason to leave them there. Hence removing them ..